templates/docs/examples/patterns/contextual-menu/_script.js

```diff
@@ -15,6 +15,15 @@ function toggleMenu(element, show, top) {
     if (typeof top !== 'undefined') {
       target.style.top = top + 'px';
     }
+
+    if (show) {
+      var focusableElements = 'button, [href], input, select, textarea, [tabindex]:not([tabindex="-1"])';
+      var firstFocusable = target.querySelector(focusableElements);
+
+      if (firstFocusable) {
+        firstFocusable.focus();
```

Comment:
Won't this be an array? I think we just need to focus on the `target` then any tabbing after that will start in the menu.